### PR TITLE
update of DBC name

### PR DESCRIPTION
**Pull request by stanvins**

### CODE REVIEW

File: src/client/components/base/T/T.json [hunk 1]
```diff
@@ -232,8 +232,8 @@
         "en": "Write to the management"
       },
       "addressCompany": {
-        "da": "DBC a/s",
-        "en": "DBC a/s"
+        "da": "DBC DIGITAL A/S",
```
Comment (slfdbc) [inline src/client/components/base/T/T.json:235]: Hedder vi ikke "DBC Digital" og ikke "DBC Digital A/S" ?

File: src/client/components/modals/ProfileModal/ProfileModal.component.js [hunk 2]
```diff
@@ -316,7 +316,7 @@ export class ProfileModal extends React.Component {
               samarbejde med DBC.
               <div>
                 <br />
-                <div>DBC as</div>
+                <div>DBC DIGITAL A/S</div>
```
Comment (slfdbc) [inline src/client/components/modals/ProfileModal/ProfileModal.component.js:319]: Igen:
Hedder vi ikke "DBC Digital" og ikke "DBC Digital A/S" ?

**Review by slfdbc — APPROVED** [review]

File: src/client/components/base/T/T.json (hunk 1 above)
Comment (stanvins) [reply src/client/components/base/T/T.json:235]: image

**Comment by stanvins** [comment]
Something has gone with the merge. I am deleting this branch and creating a new one.

**Pull request closed without merging**